static/scripts/oppia-player-0.0.0.js

```diff
@@ -208,6 +210,9 @@
     }
 
     var iframe = document.createElement('iframe');
+    iframe.isEmbedded = true;
```

Comment:
Actually, you can't change this script. It's already serving off of CDNs, so if we want to update it we'll have to make a new version. This is a bit annoying to do, so ideally we would have a solution that doesn't involve modifying anything in static/scripts at all.

To check whether the site is embedded, I recommend using the URL. See UrlService in app.js -- it has a method called isIframed().